_extras/common-issues.md

```diff
@@ -137,6 +137,30 @@ $ git config --global core.editor "nano -w"
 ~~~
 {: .language-bash}
 
+
+## SSH key authentication issues with Windows Git
+
+Rather unhelpfully, Git for Windows will use it's own SSH instance by default 
```

Comment:
This fix was suggested by one of my teammates @JezSw. Presumably there are some differences between
the two implementations because of this error arising, but I don't know the specifics. Please let me
know if you are fine to keep this around.

Reply:
Minor fixes in [1efc1a0](https://github.com/carpentries-incubator/python-intermediate-development/pull/371/commits/1efc1a0f20dfe438722e1052363bd2d518496970)

Inclusion of this section left to your decision @anenadic